pkg/util/clusteroperator/status.go

```diff
@@ -0,0 +1,65 @@
+package clusteroperator
+
+import (
+	configv1 "github.com/openshift/api/config/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SetStatusCondition returns the result of setting the specified condition in
+// the given slice of conditions.
+func SetStatusCondition(oldConditions []configv1.ClusterOperatorStatusCondition, condition *configv1.ClusterOperatorStatusCondition) []configv1.ClusterOperatorStatusCondition {
```

Comment:
fyi, I think these exist here: https://github.com/openshift/library-go/blob/master/pkg/config/clusteroperator/v1helpers/status.go

Reply:
That's based on the updated ClusterOperator so would require an openshift/api bump too. We'll grab that in the next update after we get the initial version committed.